web/api/backend.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	jwtmiddleware "github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+
+	"github.com/gorilla/mux"
+)
+
+// Claims object that will be encoded to a JWT.
+// We add jwt.StandardClaims as an embedded type, to provide fields like expiry time.
+type Claims struct {
+	UploadCredentials map[string]interface{}
+	jwt.StandardClaims
+}
+
+var jwtKey = []byte("my_secret_key")
+var creds WrapperCommand
```

Comment:
Judging by the type, 'creds' is the oauth2l command wrapper. Maybe rename it to something to what it actually represents, like 'cmd' or something like that?

Reply:
changed variable to cmd